<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kn03</class>
 <widget class="QWidget" name="Kn03">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>713</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Создание телеграммы КН-03</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Индекс станции</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="IIiii"/>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_27">
     <property name="text">
      <string>Дата и срок наблюдения</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QDateTimeEdit" name="dt">
     <property name="displayFormat">
      <string>dd.MM.yy H:mm</string>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QComboBox" name="part">
     <item>
      <property name="text">
       <string>Часть A</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Часть B</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Часть C</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Часть D</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="5">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <widget class="QPushButton" name="help">
     <property name="text">
      <string>Схема кода</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="7">
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>962</width>
        <height>582</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0" rowspan="2">
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QToolButton" name="removeBtn">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Удалить</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/meteo/icons/tools/minus.png</normaloff>:/meteo/icons/tools/minus.png</iconset>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>658</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item row="0" column="0">
            <widget class="QToolButton" name="addBtn">
             <property name="minimumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>28</width>
               <height>28</height>
              </size>
             </property>
             <property name="toolTip">
              <string>Добавить</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset>
               <normaloff>:/meteo/icons/tools/plus.png</normaloff>:/meteo/icons/tools/plus.png</iconset>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QTableWidget" name="levels">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="tabKeyNavigation">
              <bool>false</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderDefaultSectionSize">
              <number>25</number>
             </attribute>
             <column>
              <property name="text">
               <string>Тип</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Уровень, гПа</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Геопотенциал, м</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Направление ветра, °</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Скорость ветра, м/с</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="3" column="0">
            <widget class="QGroupBox" name="shearMode2">
             <property name="title">
              <string>Сдвиг ветра (второй максимум)</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>Ниже</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="vbvb2"/>
              </item>
              <item>
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>Выше</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="vava2"/>
              </item>
             </layout>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QGroupBox" name="shearMode">
             <property name="title">
              <string>Сдвиг ветра (максимум)</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Ниже</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="vbvb"/>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Выше</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="vava"/>
              </item>
             </layout>
             <zorder>label_2</zorder>
             <zorder>label_3</zorder>
             <zorder>vbvb</zorder>
             <zorder>vava</zorder>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="a4">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <item>
              <property name="text">
               <string>Прибор с датчиком давления</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Оптический теодолит</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Радиотеодолит</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Радиолокатор</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Прибор, в котором датчик давления отказал</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>VLF-Omega</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Loran-C</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Профилометр ветра</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Спутниковая навигация</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Зарезервировано</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>206</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QSplitter" name="splitter">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTextEdit" name="codeView">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QTreeWidget" name="decodeView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="MinimumExpanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <column>
           <property name="text">
            <string>Название</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Код</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Цифра кода</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Значение</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Ед. изм.</string>
           </property>
          </column>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="3" column="0" colspan="7">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="sendInMsg">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Отправить в ЦКС</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="accept">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="7">
    <widget class="QFrame" name="frame_4">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_42">
        <property name="text">
         <string>Сокращенный заголовок телеграммы в формате ВМО:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;T&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;T&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="T1T2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;1&lt;/span&gt;A&lt;span style=&quot; vertical-align:sub;&quot;&gt;2&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="A1A2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>ii</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ii">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_37">
        <property name="text">
         <string>CCCC</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="CCCC">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>0</height>
         </size>
        </property>
        <property name="frame">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_40">
        <property name="text">
         <string>BBB</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="BBB">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>612</width>
          <height>17</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <zorder>dt</zorder>
  <zorder>IIiii</zorder>
  <zorder>label</zorder>
  <zorder>label_27</zorder>
  <zorder>part</zorder>
  <zorder>frame_4</zorder>
  <zorder>help</zorder>
  <zorder>scrollArea</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
